locales/sv_SE/pcsx2_Main.po

```diff
@@ -700,14 +700,14 @@ msgid ""
 msgstr ""
 "Välj egenhändigt en ISO vid Uppstart, vilket åsidosätter valet i ISO-listan."
 
-#: pcsx2/gui/AppRes.cpp:48
-msgid "Browse for an ISO that is not in your recent history."
-msgstr "Bläddra efter en ISO som ej är i Er historielista."
-
 #: pcsx2/gui/AppRes.cpp:48
 msgid "Browse..."
 msgstr "Bläddra ..."
 
+#: pcsx2/gui/AppRes.cpp:48
+msgid "Browse for an ISO that is not in your recent history."
+msgstr "Bläddra efter en ISO som ej är i Er historielista."
```

Comment:
```suggestion
msgstr "Bläddra efter en ISO som ej är i er historik."
```
And what's the deal with 'er' and 'ni' formalities? You know that normal software translations aren't this formal right?

Reply:
Yes, but I like it (here anyway).